avocado/core/messages.py

```diff
@@ -301,3 +302,36 @@ class WhiteboardMessageHandler(BaseRunningMessageHandler):
     def handle(self, message, task, job):
         self._save_message_to_file('whiteboard', message['log'], task,
                                    message.get('encoding', None))
+
+
+class FileMessageHandler(BaseRunningMessageHandler):
+    """
+    Handler for file message.
```

Comment:
If others are not handler, is this a handler?

Reply:
No, it's mistake, All of them have to be same.